tests/weekly_tests/phys_netlist_paper.yaml

```diff
@@ -64,4 +64,4 @@ designs:
 
 flow: vivado_phys_netlist_cmp
 
-synth: "-flatten_hierarchy full -max_dsp 0"
\ No newline at end of file
+synth_options: "-flatten_hierarchy full -max_dsp 0"
```

Comment:
I think this is added by the flow already, and not required here.  Can you double check?

Reply:
This one is added by the flow already